ores/score_request.py

```diff
@@ -73,3 +73,27 @@ def __repr__(self):
                 "injection_caches={0!r}".format(self.injection_caches),
                 "ip={0!r}".format(self.ip),
                 "model_info={0!r}".format(self.model_info)]))
+
+    def toJSON(self):
+        return {
+            'context': self.context_name,
+            'rev_ids': self.rev_ids,
+            'model_names': self.model_names,
+            'precache': self.precache,
+            'include_features': self.include_features,
+            'injection_caches': self.injection_caches,
+            'ip': self.ip,
+            'model_info': self.model_info
+        }
+
+    @classmethod
+    def fromJSON(cls, kwargs):
+        return cls(
+            kwargs['context'],
+            kwargs['rev_ids'],
+            kwargs['model_names'],
+            precache=kwargs.get('precache', False),
```

Comment:
Curious why there are defaults here.  It doesn't seem like the fields are optional in our serialized format, at least.

Reply:
Indeed.  Why not just do cls(**kwargs)? 

Also, I think this is a confusing use of "kwargs" since it generally refers to key word arguments, but here, you are taking a deserialized JSON document.